plugins/woocommerce/src/Internal/Admin/Settings/PaymentsProviders/PaymentGateway.php

```diff
@@ -228,7 +228,22 @@ public function get_supports_list( WC_Payment_Gateway $payment_gateway ): array
 	 * @return bool True if the payment gateway is enabled, false otherwise.
 	 */
 	public function is_enabled( WC_Payment_Gateway $payment_gateway ): bool {
-		return filter_var( $payment_gateway->enabled, FILTER_VALIDATE_BOOLEAN );
+		try {
```

Comment:
It is better to log when dealing with a wrong type since `wc_string_to_bool` will error out if it is not a bool or a string.